### PR TITLE
feat: rename mirroring cmds to home and remote

### PR DESCRIPTION
**Pull request by fraidev**
What is changed:
- [x] `core` commands are `remote` commands now
- [x] `edge` commands are `home` commands now

### CODE REVIEW

File: crates/fluvio-cli/src/client/home/status.rs [hunk 1]
```diff
@@ -27,12 +28,26 @@ impl StatusOpt {
             .filter_map(|item| {
                 match item.spec.remote_type {
                     RemoteType::Core(core) => {
+                        let now = SystemTime::now()
+                            .duration_since(SystemTime::UNIX_EPOCH)
+                            .unwrap_or_default()
```
Comment (sehz) [inline crates/fluvio-cli/src/client/home/status.rs:33]: add unit test and also seems duplicated twice

**Review by sehz — CHANGES_REQUESTED** [review]

File: crates/fluvio-cli/src/client/home/status.rs (hunk 1 above)
Comment (fraidev) [reply crates/fluvio-cli/src/client/home/status.rs:33]: Right! About "duplicated twice" is about the home/status.rs and remote/list.rs code?

Comment (fraidev) [reply crates/fluvio-cli/src/client/home/status.rs]: I added a new method `last_seen` in RemoteStatus with tests

**Review by sehz — APPROVED** [review]
LGTM

**Pull request merged**